<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Copyright (C) 2010 Bernd Stramm</author>
 <class>ImportDialog</class>
 <widget class="QDialog" name="ImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Merge Notes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../nota.qrc">
    <normaloff>:/img/nb.png</normaloff>:/img/nb.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="importPath">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="overwriteOld">
       <property name="text">
        <string>Overwrite matching Notes</string>
       </property>
       <attribute name="buttonGroup">
        <string>newNotesGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="addNew">
       <property name="text">
        <string>All Notes are new</string>
       </property>
       <attribute name="buttonGroup">
        <string>newNotesGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="oneNewBook">
       <property name="text">
        <string>Make new Book</string>
       </property>
       <attribute name="buttonGroup">
        <string>bookGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="importMergeBooks">
       <property name="text">
        <string>Import + Match Books</string>
       </property>
       <attribute name="buttonGroup">
        <string>bookGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="showLog">
       <property name="text">
        <string>Show Log</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start Merge</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../nota.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="newNotesGroup"/>
  <buttongroup name="bookGroup"/>
 </buttongroups>
</ui>
